xla/service/float_normalization_test.cc

```diff
@@ -500,6 +500,36 @@ TEST_F(FloatNormalizationTest, DoNotChangeBitcastConvert) {
   EXPECT_EQ(root->operand(0)->shape().element_type(), U16);
 }
 
+TEST_F(FloatNormalizationTest, ResolveIfUnsupportedF8e4m3) {
```

Comment:
Merge this with the existing test ResolveIfUnsupportedF8e5m2, either by looping over values (F8E4M3, F8E5M2) or by using a value-parameterized test with TEST_P.

Reply:
Opened PR https://github.com/openxla/xla/pull/17177 - Parametrize FloatNormalizationF8Test ResolveIfUnsupportedF8